ladybug/datacollection.py

```diff
@@ -410,7 +483,7 @@ def interpolate_data(self, timestep, cumulative=False):
                 _data[i].value = d.value / timestep
 
         # shift data if half-hour interpolation has been selected.
-        if self.header.middle_hour is True:
+        if self.header.data_type.middle_hour_epw is True:
```

Comment:
I still think we should find a better term for `middle_hour_epw`. Not sure what would be a good term but this will come back to bite us later.

Reply:
I agree that it's better to invest time now in coming up with a good name rather than going through the pain of changing it later.

I really wish that the "cumulative" boolean that I put on the DataType could have taken care of this since the EPW documentation originally said that the reason why solar radiation occurs on the half hour is because it represents an accumulation of solar energy over the previous hour.  And then they go and throw this reasoning to the wind by taking a data type that is not cumulative but averaged over time (illuminance) and applying the same logic.

Maybe it would be easier to label this data with what it is not (and just reverse the boolean).  For example, this property could be called "point_in_time" to denote the fact that most measurements in the EPW occur on the hour at that point in time.  The only exceptions are radiation, which is cumulative over time, and illuminance, which is averaged over time.

This could also be useful for differentiating between many of the results from E+ that are an average or accumulation over timesteps rather than an exact value at the timestep.  I'll let you know if I come up with anything else but I am starting to think that "point_in_time" might be the best option.

Reply:
I just changed this to point_in_time and I think it is a lot clearer now.

Reply:
`point_in_time` should do it! Thank you for the change.